azurerm/internal/services/netapp/netapp_volume_resource.go

```diff
@@ -194,6 +194,43 @@ func resourceNetAppVolume() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+
+			"data_protection_replication": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				ForceNew: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"endpoint_type": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Default:  "dst",
+							ValidateFunc: validation.StringInSlice([]string{
+								"dst",
+							}, true),
```

Comment:
could we not be case insensitive here
```suggestion
							}, falsee),
```

Reply:
I thought I changed it.